src/components/CreateNewBudgetSheet.js

```diff
@@ -0,0 +1,177 @@
+import { useState } from 'react';
+import firebase from '../firebase';
+import { getDatabase, ref, push } from 'firebase/database';
+import { useNavigate } from 'react-router-dom';
+import WarningModal from './WarningModal';
+
+const NewBudgetSheet = (props) => {
+  const database = getDatabase(firebase);
+  const dbRef = ref(database);
+
+  const [name, setName] = useState('');
+  const [budgetTotal, setBudgetTotal] = useState('');
+  const [categoriesInput, setCategoriesInput] = useState('');
+  const [categoriesArray, setCategoriesArray] = useState([]);
+  const [isWarning, setIsWarning] = useState(false);
+
+  // Forms
+  const handleNameInputChange = (e) => {
+    setName(e.target.value);
+  };
+
+  const handleBudgetInputChange = (e) => {
+    setBudgetTotal(e.target.value);
+  };
+
+  let navigate = useNavigate();
+
+  const handleSubmit = (e, budgetAmount, nameOfSheet, categories) => {
```

Comment:
These arguments appear in a different order than the associated inputs do on the page.

Reply:
fixed